libyul/backends/wasm/TextTransform.cpp

```diff
@@ -41,14 +44,23 @@ string TextTransform::run(wasm::Module const& _module)
 	string ret = "(module\n";
 	for (auto const& sub: _module.subModules)
```

Comment:
Changed, but I bet there are thousands of similar cases in the codebase -- do you want to create a tracking issue for the "reduce technical debt" project?

Reply:
Nah, I don't think that wholesale refactors like that are a good idea in case of a tiny cleanup like this. They're almost impossible to review properly. In my opinion it's best to improve smaller pieces of code as they get rewritten or updated and this is just such a case.

Also, it's just a suggestion, not a deal breaker. I'd accept the PR anyway if you said you don't want to change it :)